src/main/java/org/reactome/release/downloaddirectory/GenerateGOAnnotationFile/GOAGeneratorUtilities.java

```diff
@@ -186,46 +245,51 @@ public static Integer assignDateForGOALine(GKInstance entityInst, String goaLine
         return instanceDate;
     }
 
+    static String getTaxonIdentifier(GKInstance protein) throws Exception {
```

Comment:
Why not public/private scope instead of package scope?

Reply:
This method is invoked in the test code, so package scope is the most restricted it can be.

Reply:
Does it need to be invoked directly, or could it be tested by calling a method that calls it?

Reply:
The method isn't itself being tested - it is being mocked (since it is a utils method) for methods being tested for other classes.